task_processing/plugins/mesos/mesos_executor.py

```diff
@@ -14,6 +14,7 @@
     ExecutionFramework
 )
 from task_processing.plugins.mesos.translator import mesos_status_to_event
+from task_processing.plugins.smartstack import get_mesos_master
```

Comment:
i would just make smartstack module part of mesos plugin. as implemented, `task_processing.plugins.smartstack` does nothing without mesos plugin and mesos plugin would fail to work without smartstack module due to this import.